pyproject.toml

```diff
@@ -1,6 +1,45 @@
-[build-system]
-requires = [
-    "setuptools>=42",
-    "wheel"
+[tool.poetry]
+name = "momento"
+version = "0.0.0"
+
+authors = ["Momento <open-source@momentohq.com>"]
+description = "SDK for Momento"
+
+license = "Apache-2.0"
+
+documentation = "https://docs.momentohq.com/"
+homepage = "https://gomomento.com"
+readme = "README.md"
+repository = "https://github.com/momentohq/client-sdk-python"
+
+keywords = ["Momento", "caching", "key-value store"]
```

Comment:
maybe add serverless to this

Reply:
Done in [1beff1f](https://github.com/momentohq/client-sdk-python/pull/133/commits/1beff1f2d5509f0bbeed85c227817c0ff3372b16)